test/recipes/30-test_evp_data/evpmac.txt

```diff
@@ -131,23 +131,29 @@ Output = 5150d1772f50834a503e069a973fbd7c
 # SIPHASH - default values: 2,4 rounds, explicit 8-byte mac
 
 MAC = SipHash
-Ctrl = digestsize:8
+#In 3.0 this ctrl seems to have been renamed to "size"
```

Comment:
Perhaps we should ensure that there's an alias in 3.0...  Not gonna block this change, though...

Reply:
This is a bug in 3.0 IMO. It's being tracked through https://github.com/openssl/openssl/issues/14370, this is a temporary workaround to make the tests pass until we fix the bug.